### DISCUSSION TITLE
Plans for the datepicker

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#45405] [General]
Hi there,

Just a question here, are there any future plan to do with datepicker? e.g making it more mobile friendly etc?

thanks

**Comment by zombieJ** [comment]
Yes. Recent I've collected related issue. and check if we have to make it better experience.

**Reply by Saidnazar08** [reply]
Hello @zombieJ.
I want to help you guys with this. What is your plan to solve this problem? I want to create a PR for the Trigger component and am going to add a "centered" boolean property to it that, if true, displays a popup in the middle of the screen. Even cooler is if we use the isMobile utility to display a popup in the middle of the screen.

**Reply by yoyo837** [reply]
The team plans to completely reimplement DatePicker and related components.

**Reply by Saidnazar08** [reply]
Oh, this is going to take a long time, isn't it? When are you planning to reimplement and release it?

**Reply by AsuraKev** [reply]
How exciting! thanks Antd team :) hopefully its not too far away
